runners/maven/src/main/scala/stryker4s/run/MavenMutantRunner.scala

```diff
@@ -0,0 +1,56 @@
+package stryker4s.run
+import java.nio.file.Path
+import java.util.Properties
+
+import better.files._
+import org.apache.maven.project.MavenProject
+import org.apache.maven.shared.invoker.{DefaultInvocationRequest, InvocationRequest, Invoker}
+import stryker4s.config.Config
+import stryker4s.model.{Killed, Mutant, MutantRunResult, Survived}
+import stryker4s.mutants.findmutants.SourceCollector
+
+import scala.collection.JavaConverters._
+
+class MavenMutantRunner(project: MavenProject, invoker: Invoker, sourceCollector: SourceCollector)(
+    implicit config: Config
+) extends MutantRunner(sourceCollector) {
+
+  private val goals = List("test").asJava
+
+  private val properties = new Properties(project.getProperties)
+  properties.setProperty("surefire.skipAfterFailureCount", 1.toString) // Stop after first failure. Only works with surefire plugin, not scalatest
+
+  override def runInitialTest(workingDir: File): Boolean = {
+    // Set source once, settings is persistent among goals
+    invoker.setWorkingDirectory(workingDir.toJava)
```

Comment:
putt this on class level? doesn't make sense to do it initialtestRun if it's persistent. If we would ever disable the initial test run this would break the mutation run